src/kixi/datastore/schemastore/validator.clj

```diff
@@ -54,6 +54,10 @@
   [definition _]
   (conformers/regex? (get-in definition [::ss/schema ::ss/pattern])))
 
+(defmethod resolve-schema "string"
+  [_ _]
+  conformers/-string?)
```

Comment:
How is this even allowing you to call a private fn? Either way,  probably best to lose the `-`